.eslintrc.js

```diff
@@ -0,0 +1,24 @@
+module.exports = {
+  env: {
+    browser: true,
+    commonjs: true,
+    es2021: true,
+  },
+  extends: [
+    'plugin:react/recommended',
+    'airbnb',
+  ],
+  parserOptions: {
+    ecmaFeatures: {
+      jsx: true,
+    },
+    ecmaVersion: 'latest',
+  },
+  plugins: [
+    'react',
+  ],
```

Comment:
Why react plugin since you're on the backend? 
PLease you don't need to add react plugin on the backend as to work will require you to have that plugin in your project.
Consider removing react as plugin, again you're allowing `jsx` on the backend? 
Differentiate `nodejs linting` and `react linting`